model-events/src/main/java/de/focus_shift/urlaubsverwaltung/extension/api/application/VacationTypeDTO.java

```diff
@@ -0,0 +1,16 @@
+package de.focus_shift.urlaubsverwaltung.extension.api.application;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class VacationTypeDTO {
+  @NonNull private String category;
```

Comment:
Is a vacationTypeDto only distinguished from others by its category?

Reply:
right now, yes. but it was a controversial discussion. with it's own model, we are ready for extending with more attributes without breaking the model.

having just a String named vacationTypeCategory the model is not extendable and closed for enhancements